tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "usage: $0 <repository-root> <tool-name> [...]"
```

Comment:
What is `[...]` here? It doesn't seem to be used in the script, or described in this usage doc.